platform/pwl.c

```diff
@@ -699,17 +708,17 @@ keyboard_on_keymap (void *data,
         return;
     }
 
-    g_clear_pointer (&display->xkb_data.state, xkb_state_unref);
-    g_clear_pointer (&display->xkb_data.keymap, xkb_keymap_unref);
+    g_clear_pointer (&seat->xkb_data.state, xkb_state_unref);
+    g_clear_pointer (&seat->xkb_data.keymap, xkb_keymap_unref);
 
-    display->xkb_data.keymap = g_steal_pointer (&keymap);
-    display->xkb_data.state = g_steal_pointer (&state);
+    seat->xkb_data.keymap = g_steal_pointer (&keymap);
+    seat->xkb_data.state = g_steal_pointer (&state);
 
-    display->xkb_data.indexes.control = xkb_keymap_mod_get_index (display->xkb_data.keymap,
+    seat->xkb_data.indexes.control = xkb_keymap_mod_get_index (seat->xkb_data.keymap,
                                                                   XKB_MOD_NAME_CTRL);
```

Comment:
Plese reindent the dangling lines so they are aligned to the opening parenthesis:

```c
xkb_keymap_mod_get_index (seat->xkb_data.keymap,
                          XKB_MOD_NAME_CTRL);
           // align here ↑
```

Same thing in the lines below.

Reply:
done